packages/decision-tree/src/components/Question/questionTypes/MultipleChoice/MultipleChoice.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, useState } from 'react';
+import { type QuestionProps } from '@components/Question/types';
+import { Box, Checkbox } from '@cmpsr/components';
+import { QuestionTitle } from '../../components/QuestionTitle';
+import { MultipleChoiceQuestion } from './types';
+import { HandleAnswersActions } from '@hooks';
+
+export const MultipleChoice: FC<QuestionProps> = ({ data, answersDispatch }) => {
+  const { question, choices, tooltip } = data as MultipleChoiceQuestion;
+  const [answers, setAnswers] = useState(new Set<string>());
+
+  return (
+    <Box>
```

Comment:
ditto

Reply:
can we approach this in the next PR where all the styles are applied?